lib/rules/wrap-iife.js

```diff
@@ -46,64 +58,95 @@ module.exports = {
                 nextToken && nextToken.value === ")";
         }
 
-        return {
+        /**
+        * Get the function node from an IIFE
+        * @param {ASTNode} node to evaluate
+        * @returns {ASTNode} node that is the function expresion of the given IIFE, or null if non exist
+        */
+        function getFunctionNodeFromIIFE(node) {
+            if (node.type !== "CallExpression") {
```

Comment:
Will this ever be the case? Doesn't seem necessary since it looks like this function is only called once and we know for sure that the node will be a `CallExpression` node